### PR TITLE
testing/perl-perlmagic: new aport

### PR DESCRIPTION
**Pull request by timlegge**
@ncopa as you are the maintainer of the ImageMagick aport it would likely be a good idea for you to review this perlmagick aport. 

It needs ImageMagick 7.0.8.36 packages to be updated as well.

### CODE REVIEW

**Comment by Ikke** [comment]
Please adjust the commit message to the [aports style][0], for example: 

> testing/perl-permagic: new aport

[0]:https://github.com/alpinelinux/aports/blob/master/.github/CONTRIBUTING.md#creating-a-pull-request-pr

File: testing/perlmagick/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+	chrpath -d ./blib/arch/auto/Image/Magick/Q16HDRI/Q16HDRI.so
+}
+
+check() {
+	cd "$builddir/PerlMagick"
+	if ! [ -e blib/lib/Image/Magick.pm ]; then
+		error "Module does not exist Magick.pm"
+		return 1
+	fi
+        if ! [ -e blib/lib/Image/Magick/Q16HDRI.pm ]; then
```
Comment (Ikke) [inline testing/perlmagick/APKBUILD]: Inconsistent whitespace. Please use tabs for indentation.

File: testing/perlmagick/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+_pkgver=${pkgver%.*}-${pkgver##*.}
+_abiver=7
+pkgdesc="Perl Module for Image Magick"
+url="https://www.imagemagick.org/"
+arch="all"
+license="ImageMagick"
+options="libtool !checkroot"
+depends="imagemagick imagemagick-libs imagemagick-c++"
+makedepends="perl-dev libtool chrpath"
+checkdepends=""
```
[the remark this answers is not in the document]
Comment (timlegge) [reply testing/perlmagick/APKBUILD]: Will Do

File: testing/perlmagick/APKBUILD (hunk 1 above)
Comment (timlegge) [reply testing/perlmagick/APKBUILD]: oops.  Will do

**Comment by timlegge** [comment]
will squash the changes and send a force push

**Comment by Ikke** [comment]
@timlegge Why does ImageMagick need to be updated? This seems a new package. And if that's the cause it would be best to do it in the same PR, so it can be applied in one go.

Thanks.

**Comment by timlegge** [comment]
> @timlegge Why does ImageMagick need to be updated? This seems a new package. And if that's the cause it would be best to do it in the same PR, so it can be applied in one go.

perlmagick is the perl modules to use ImageMagick via perl.  It essentially requires adding the --with-perl to the configure for ImageMagick and doing the make of the perl modules and perlmagick install.  I did not see a way to do that with the ImageMagick (like a sub package).  Many distributions consider it to be a separate installation package you may want imagemagick or both imagemagick and perlmagick

It does however require the imagemagic and the permagick imagemagick versions to be the same.

If you can think of alternatives let me know.

**Comment by timlegge** [comment]
I will likely close this PR as I have figured out how to add it to the main/imagemagick package

**Comment by timlegge** [comment]
I have created PR7078 https://github.com/alpinelinux/aports/pull/7078 which would appear to be the best way to address this package for perlmagick

**Comment by Ikke** [comment]
Would it make sense to close this PR then?

**Comment by timlegge** [comment]
yes.  closing

**Pull request closed without merging**